ansible/roles/ansible-horizon-plugin/tasks/post_install.yml

```diff
@@ -1,42 +1,74 @@
+  when: ENV_PATH != "/usr"
+
+- block:
+  - name: copy tvault_panel_group.py file to remote location
+    copy: src="{{automation_home}}/files/tvault_panel_group.py" dest="{{HORIZON_PATH}}/openstack_dashboard/local/enabled/tvault_panel_group.py"
+    notify:
+       - restart webserver
+
+  - name: copy tvault_admin_panel_group.py file to remote location
+    copy: src="{{automation_home}}/files/tvault_admin_panel_group.py" dest="{{HORIZON_PATH}}/openstack_dashboard/local/enabled/tvault_admin_panel_group.py"
+    notify:
+       - restart webserver
 
-- name: copy tvault_filter.py file to remote location
-  copy: src="{{automation_home}}/files/tvault_filter.py" dest="{{HORIZON_PATH}}/openstack_dashboard/templatetags/tvault_filter.py"
+  - name: copy tvault_panel.py file to remote location
+    copy: src="{{automation_home}}/files/tvault_panel.py" dest="{{HORIZON_PATH}}/openstack_dashboard/local/enabled/tvault_panel.py"
+    notify:
+       - restart webserver
+
+  - name: copy tvault_settings_panel.py file to remote location
+    copy: src="{{automation_home}}/files/tvault_settings_panel.py" dest="{{HORIZON_PATH}}/openstack_dashboard/local/enabled/tvault_settings_panel.py"
+    notify:
+       - restart webserver
+
+  - name: copy tvault_admin_panel.py file to remote location
+    copy: src="{{automation_home}}/files/tvault_admin_panel.py" dest="{{HORIZON_PATH}}/openstack_dashboard/local/enabled/tvault_admin_panel.py"
+    notify:
+       - restart webserver
+
+  - name: copy tvault_filter.py file to remote location
+    copy: src="{{automation_home}}/files/tvault_filter.py" dest="{{HORIZON_PATH}}/openstack_dashboard/templatetags/tvault_filter.py"
+    notify:
+       - restart webserver
+  when: >
+           (ansible_distribution_major_version=="7" and ansible_distribution=="CentOS") or
+           (ansible_distribution_major_version=="7" and ansible_distribution=="RedHat") or
+           (ansible_distribution_major_version=="8" and ansible_distribution=="CentOS") or
+           (ansible_distribution_major_version=="8" and ansible_distribution=="RedHat")
+
+- name: Copy Trilio Dashboard Files when Ubuntu
+  shell: |
+      cp -rf /usr/share/openstack-dashboard/openstack_dashboard/local/enabled/*panel*.py {{HORIZON_PATH}}/openstack_dashboard/local/enabled/
+      cp -f /usr/share/openstack-dashboard/openstack_dashboard/templatetags/tvault_filter.py {{HORIZON_PATH}}/openstack_dashboard/templatetags/
   notify:
-     - restart webserver
-
-- name: write content into destination file - sync_static.py 
-  copy: src="{{automation_home}}/files/sync_static.py" dest="/tmp/sync_static.py" 
- 
-- name: Change the working directory to horizon and excute shell command
-  shell: "{{ MANAGE_FILE }} shell < /tmp/sync_static.py &> /dev/null"
-  args:
-       chdir: "{{HORIZON_PATH}}"
-
-- name: remove temporary file -  sync_static.py
-  file: path=/tmp/sync_static.py state=absent
+    - restart webserver
+  when: ansible_distribution=="Ubuntu"
```

Comment:
these file copying should get handled by the packaging. 
copy will happen only if there is a virtualenv activated for horizon.

Reply:
As LXC container using virtual env